<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="softwaretestingmaterial" duration-ms="39550" started-at="2020-03-15T12:38:41Z" finished-at="2020-03-15T12:39:21Z">
    <groups>
    </groups>
    <test name="testngTest" duration-ms="39550" started-at="2020-03-15T12:38:41Z" finished-at="2020-03-15T12:39:21Z">
      <class name="com.iNetbanking.testcases.TC_Logintest_001">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.iNetbanking.testcases.TC_Logintest_001@6093dd95]" name="setUp" is-config="true" duration-ms="15979" started-at="2020-03-15T12:38:41Z" finished-at="2020-03-15T12:38:57Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="logIn()[pri:0, instance:com.iNetbanking.testcases.TC_Logintest_001@6093dd95]" name="logIn" duration-ms="9" started-at="2020-03-15T12:38:57Z" finished-at="2020-03-15T12:38:57Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.iNetbanking.testcases.TC_Logintest_001.logIn(TC_Logintest_001.java:12)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.iNetbanking.testcases.TC_Logintest_001@6093dd95]" name="tearDown" is-config="true" duration-ms="1178" started-at="2020-03-15T12:38:57Z" finished-at="2020-03-15T12:38:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.iNetbanking.testcases.TC_Logintest_001 -->
      <class name="com.iNetbanking.testcases.TC_HeaderLogoDisplayed_002">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.iNetbanking.testcases.TC_HeaderLogoDisplayed_002@5622fdf]" name="setUp" is-config="true" duration-ms="14918" started-at="2020-03-15T12:38:58Z" finished-at="2020-03-15T12:39:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="headerlogos()[pri:0, instance:com.iNetbanking.testcases.TC_HeaderLogoDisplayed_002@5622fdf]" name="headerlogos" duration-ms="6738" started-at="2020-03-15T12:39:13Z" finished-at="2020-03-15T12:39:20Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.iNetbanking.testcases.TC_HeaderLogoDisplayed_002.headerlogos(TC_HeaderLogoDisplayed_002.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- headerlogos -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.iNetbanking.testcases.TC_HeaderLogoDisplayed_002@5622fdf]" name="tearDown" is-config="true" duration-ms="688" started-at="2020-03-15T12:39:20Z" finished-at="2020-03-15T12:39:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.iNetbanking.testcases.TC_HeaderLogoDisplayed_002 -->
    </test> <!-- testngTest -->
  </suite> <!-- softwaretestingmaterial -->
</testng-results>
